accname/name/comp_tooltip.html

```diff
@@ -13,37 +13,54 @@
 
 <p>Tests the <a href="https://w3c.github.io/accname/#comp_tooltip">#comp_tooltip</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
 
-<a href="#" title="label" data-expectedlabel="label" data-testname="link with img with tooltip label" class="ex"><img alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw=="></a>
-<a href="#" title="label" data-expectedlabel="label" data-testname="link with text with tooltip label" class="ex">content</a>
-<div title="label" role="group" data-expectedlabel="label" data-testname="div with text with tooltip label" class="ex">content</div>
-<img src="foo.jpg" title="label" data-expectedlabel="label" data-testname="img with tooltip label without alt" class="ex">
-<img src="foo.jpg" title="label" data-expectedlabel="label" alt="" data-testname="img with tooltip label with empty alt" class="ex">
-<img src="foo.jpg" title="label" data-expectedlabel="broken" alt="broken" data-testname="img with tooltip label with alt" class="ex">
-<img src="foo.jpg" data-expectedlabel="broken" alt="broken" data-testname="img with tooltip label without title" class="ex">
-
-<textarea title="enter text" data-expectedlabel="enter text" placeholder="textarea" data-testname="textarea with placeholder and tooltip label" class="ex"></textarea>
-<input type="text" title="label" data-expectedlabel="label" placeholder="text" data-testname="text input with placeholder and tooltip label" class="ex">
-<input type="email" title="label" data-expectedlabel="label" placeholder="email" data-testname="email input with placeholder and tooltip label" class="ex">
-<input type="password" title="label" data-expectedlabel="label" placeholder="password" data-testname="password input with placeholder and tooltip label" class="ex">
-<input type="checkbox" title="label" data-expectedlabel="label" data-testname="checkbox input with tooltip label" class="ex">
-<input type="radio" title="label" data-expectedlabel="label" data-testname="radio input with tooltip label" class="ex">
-<button title="label" data-expectedlabel="label" data-testname="button with tooltip label" class="ex">button</button>
-<abbr title="Web Platform Tests" data-expectedlabel="Web Platform Tests" data-testname="abbr with tooltip label" class="ex">WPT</abbr>
-<kbd title="Control + Option" data-expectedlabel="Control + Option" data-testname="kbd with tooltip label" class="ex">CTRL + OPT</kbd>
+<a href="#" title="title" data-expectedlabel="title" data-testname="link with img with tooltip label" class="ex"><img alt="" src="data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw=="></a>
```

Comment:
Reviewers, this diff is more difficult to read than anticipated, but you can see from the results diffs that not much has changed besides errata correction:

- [Safari](https://wpt.fyi/results/accname/name/comp_tooltip.html?diff&filter=ADC&run_id=5134590500405248&run_id=6195170162245632)
- [Firefox](https://wpt.fyi/results/accname/name/comp_tooltip.html?diff&filter=ADC&run_id=5143970708979712&run_id=6314389294022656) (now more obvious it's failing the placeholder vs title expectations in HTML-AAM)
- [Chrome](https://wpt.fyi/results/accname/name/comp_tooltip.html?diff&filter=ADC&run_id=4908487215153152&run_id=5068241845616640)